packages/core/src/server/service.ts

```diff
@@ -124,6 +131,33 @@ export class ServerService {
       }
     });
 
+    // TODO: Only register in development mode.
+    this.app.post("/admin/reload", async (req, res) => {
+      try {
+        const chainIdsQuery = req.query.chainIds ?? "31337";
```

Comment:
1. What would this look like as a curl request?
2. I think it makes sense to response with a validation error if the user does not include a chain ID rather than default. Not specifying one seems like a "bad habit".

Reply:
1. 
`curl -X POST "localhost:42069/admin/reload?chainId=31337"`

That would be to reload `anvil`'s default chain ID.

2. I agree, I ended up removing it :)